calico-cloud/image-assurance/scanners/registry-scanner.mdx

```diff
@@ -0,0 +1,150 @@
+---
+description: Scan images in container registeries.
+---
+
+# Scan images in container registries
+
+Scan images in container registries at any time, on any infrastructure, including Kubernetes.
+
+## Value
+
+Add a layer of defense for images that don’t go through a pipeline (for example, third-party images), but are published to registries. If CVEs are missed in your build pipeline, you can catch them before they are deployed.
+
+## Concepts
+
+You can deploy the registry scanner wherever you want. Locally on a laptop. Anywhere. And it doesn’t have to be for Kubernetes.
+
+To use this option, all you need to do is:
+- Specify the registry paths to the images you want to scan
+- Provide permissions for the scanner to access your registries
+- Get a token for access to the Image Assurance API
+
+Based on the paths you specify, the scanner recursively scans all images in the registry once, and sends results to the Image Assurance dashboard in Manager UI.
+
+## Before you begin
+
+**Required**
+
+- Valid registry credentials to set up handshake with the registry scanner
+
+**Supported registry platforms**
+
+- Amazon ECR
+- Azure Container Registry
+- Docker, using a container runtime
+
+**Limitations**
+
+- The registry scanner is available as an image (not using Tigera operator installation)
+- You can scan images from only one of the supported registry platforms/account per instance
+- If you use the registry scanner with Docker, only tagged images are scanned. However, if you use the scanner with Amazon AWS or Azure, all images (tagged and untagged) are scanned.
+
+## How to
+
+- [Download the registry scanner](#download-the-registry-scanner)
+- [Set up registry scanner](#set-up-registry-credentials)
+- [Scan images and send results to Calico Cloud](#scan-images-and-send-results-calico-cloud)
+- [Troubleshoot](#troubleshoot)
+- [Get registry scanner updates](#get-registry-scanner-updates)
+
+### Download the registry scanner
+
+The registry scanner comes in a Docker image. To get the image, run this command: `docker pull quay.io/tigera/image-assurance-registry-scanner:v1.10.4`.
```

Comment:
This needs to use the `imageassuranceversion` variable so that the version is up-to-date from release to release.